backend/server/mailing.go

```diff
@@ -2,91 +2,239 @@ package main
 
 import (
 	"net/http"
+	"time"
 
 	"github.com/labstack/echo/v4"
+	"github.com/mailjet/mailjet-apiv3-go"
 )
 
-// Message - struct to contain email message data
-type Message struct {
+// Enquiry - struct to contain email enquiry data
+type Enquiry struct {
 	Name  string `validate:"required"`
 	Email string `validate:"required,email"`
 	Body  string `validate:"required"`
 }
 
 // Feedback - struct to contain feedback message data
-// name is not required, email must be valid and body is required.
+// name is not required, email must be valid (or empty) and body is required.
 type Feedback struct {
```

Comment:
would it be possible to combine this struct with the Enquiry one? let me know :)

Reply:
The only reason why I wouldn't do that is that even though they have the same fields, their validation is different.